data_diff/databases/presto.py

```diff
@@ -38,7 +37,23 @@ class Presto(Database):
     def __init__(self, **kw):
         prestodb = import_presto()
 
-        self._conn = prestodb.dbapi.connect(**kw)
+        if kw.get("schema"):
+            self.default_schema = kw.get("schema")
+
+        if (
+            "password" in kw and "auth" in kw and kw.get("auth") == "basic"
+        ):  # if auth=basic, add basic authenticator for Presto
+            kw["auth"] = prestodb.auth.BasicAuthentication(kw.get("user"), kw.get("password"))
+            kw.pop("password")
+
+        if "cert" in kw:  # if a certificate was specified in URI, verify session with cert
+            cert = kw.get("cert")
+            kw.pop("cert")
+            self._conn = prestodb.dbapi.connect(**kw)
+            self._conn._http_session.verify = cert
```

Comment:
Can you please confirm that this PR implements only the old-style password authentication but with the custom HTTPS-enabled server? I.e. no _client certificates_ are supported at the moment. I initially misread the PR description as if it implements the client-side certs (similar to Snowflake's logic, I think).